src/transformers/generation/logits_process.py

```diff
@@ -1908,6 +1908,14 @@ def __init__(
     ):  # support for the kwargs
         self.no_timestamps_token_id = generate_config.no_timestamps_token_id
         self.timestamp_begin = generate_config.no_timestamps_token_id + 1
+
+        if hasattr(generate_config, "number_timestamp_tokens"):
+            number_timestamp_tokens = generate_config.number_timestamp_tokens
+        else:
+            # Whisper uses by default 1501 timestamp tokens, from <|0.00|> to <|30.00|> with a step of 20ms
+            number_timestamp_tokens = 1501
```

Comment:
Might be easier to do so ?
```suggestion
        # Whisper uses by default 1501 timestamp tokens, from <|0.00|> to <|30.00|> with a step of 20ms
        number_timestamp_tokens = generate_config.get("number_timestamp_tokens", 1501)
```

Reply:
It indeed would be! Unfortunately, `GenerationConfig` does not inherit from `dict` and therefore does not implement the `get` method.